etc/.golangci.yaml

```diff
@@ -3,11 +3,8 @@ service:
 run:
   deadline: 900s
   modules-download-mode: readonly
+  skip-dirs-use-default: false
   skip-dirs:
-    - genfiles$
```

Comment:
just for my edification - does this mean we're lining the genfiles too now, or are they begin ignored elsewhere?

Reply:
We don't have "genfiles" in RDK anymore. Generated files from protobuf live in API now.

Reply:
right! thanks!